.eslintrc.json

```diff
@@ -1,23 +1,22 @@
 {
-    "env": {
-      "browser": true,
-      "es6": true,
-      "jest": true
-    },
-    "parser": "babel-eslint",
-    "parserOptions": {
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "extends": ["airbnb-base"],
-    "rules": {
-      "no-shadow": "off",
-      "no-param-reassign": "off",
-      "eol-last": "off",
-      "arrow-parens": "off"
-    },
-    "ignorePatterns": [
-      "dist/",
-      "build/"
-    ]
-}
\ No newline at end of file
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off"
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
Thank you for making this change :+1: 

- [ ] I realized that you do not understand the source of the second issue raised by the last reviewer. So I decided to do a quick recording of what is actually causing the error, why you're not having the error at your end, and how you can possibly solve the problem. Kindly check it out [here](https://www.loom.com/share/b66960f36c824610854e5b2540a596bc)

If you sill do not understand or you're having some doubts or you need help going further, kindly request for help [here](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-)

Reply:
I want to thank you for the explanation of the video.